meetings/tool/src/main/resources/main_eu.properties

```diff
@@ -0,0 +1,10 @@
+create_new_meeting=Create new meeting
```

Comment:
@pablosanroman would you mind contributing a translation? Thanks!

Reply:
Hi @mpellicer 

create_new_meeting= Sortu bilera
error_load_meetings= Ezin izan dira bilerak kargatu
error_load_permissions= Ezin izan dira baimenak kargatu
futuro= etorkizuna
no_meetings_scheduled= Ez dago bilerarik programatuta gune honetarako.
no_results= Ez dago emaitzarik bilaketa honetarako.
pasado= Iragana
search_results= Bilaketaren emaitzak
search= Bilatu bilerak
hoy = gaur